README.md

```diff
@@ -17,26 +23,191 @@ The SDK handles the device data collection, communication with the card issuer,
 5. Meets requirements from **EMVCo and the PCI Security Standards Council**, specifically set for 3DS SDKs, so you can be sure it is interoperable with card issuers and that your customers’ sensitive data stays secure.
 
 ## Minimum Requirements
-The 3DS SDK for iOS requires Xcode 13.4 and above with Swift version 5.6.1 and above, and supports apps targeting iOS 12.0 and above. It also supports Objective-C.
+The 3DS SDK for iOS requires Xcode 13.1 and above with Swift version 5.6.2 and above, and supports apps targeting iOS 12.0 and above. It also supports Objective-C. There are two ways to integrate into our 3DS SDK.
 
 ## Getting Started
 
 First, integrate the 3DS SDK into your app.
 
+### Integrating with CocoaPods
+[CocoaPods](http://cocoapods.org) is the traditional dependency manager for Apple projects. We still support it, but we're not always able to validate all its peculiar ways.
+
+Make sure cocoapods is installed on your machine by running
+```bash
+$ pod --version
+```
+Any version newer than **1.10.0** is a good sign. If not installed, or unsupported, follow [Cocoapods Getting Started](https://guides.cocoapods.org/using/getting-started.html)
+
+Once Cocoapods of a valid version is on your machine, to integrate Frames into your Xcode project, update your `Podfile`:
+```ruby
+platform :ios, '12.0'
+use_frameworks!
+
+target '<Your Target Name>' do
+  pod 'JOSESwift', '2.2.1', :inhibit_warnings => true
```

Comment:
I'm not aware of the challenges or the reasons that we require the merchants to include this into their `Podfile`s but could this have been a dependency of our SDK?